lambdas/handlers/bulk_upload_metadata_handler.py

```diff
@@ -0,0 +1,91 @@
+import csv
+import logging
+import os
+import tempfile
+from typing import Iterable
+
+import pydantic
+from botocore.exceptions import ClientError
+from models.staging_metadata import (METADATA_FILENAME, NHS_NUMBER_FIELD_NAME,
+                                     MetadataFile, StagingMetadata)
+from services.s3_service import S3Service
+from services.sqs_service import SQSService
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def lambda_handler(_event, _context):
+    try:
+        logger.info("Starting metadata reading process")
+
+        staging_bucket_name = os.environ["STAGING_STORE_BUCKET_NAME"]
```

Comment:
Should these not be checked by the ensure_env_var decorator that @joefong-nhs has written previously?

Reply:
We thought of using that, but that decorator will respond with 500 status, and this lambda ins't going to be called from a web API, so finally we choose to just wrap this in a try block and catch KeyErrors